introduction/07-faq.asciidoc

```diff
@@ -0,0 +1,15 @@
+== FAQ
+
+To conclude the Introduction segment of the Learning Path, here are some Frequently Asked Questions people have when embarking on their InnerSource journey.
+
+=== What is the cost/overhead?
+
+=== Why not just open source?
```

Comment:
By all means do so if the project makes sense!  Some projects are specific to your company or are a competitive advantage, so you'll want to keep those as InnerSource.  Some need to iterate more quickly than can be done in the open.

Reply:
There's also the InnerSource as a stepping stone to open source angle, for teams who need to up skill these practices.

Reply:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/465/commits/0e4a7d0a88dfae20331af1a86b47e3b1ba80eef2